app/views/conference/_all_days.xml.haml

```diff
@@ -0,0 +1,3 @@
+- @conference.program.events.group_by{ |event| event.start_time.to_date }.each_with_index do | day, index |
```

Comment:
`@conference.program.events.scheduled` assuming that we only want to show scheduled events in here.